lib_json/src/main/x/json/JsonMergePatch.x

```diff
@@ -51,7 +50,7 @@ class JsonMergePatch(Doc patch) {
                     if (Doc targetValue := target.get(key)) {
                         merge(key, merge(targetValue, value, inPlace));
                     } else {
-                        merge(key, merge(json.newObject(), value, True));
+                        merge(key, merge(json.newObject(), value, True)); // TODO JK: why not "inPlace"
```

Comment:
Here we are merging `value` into a new `JsonObject` so we may as well do it in place (hence the `True` argument. There is no point doing a copy if the `inPlace` parameter is `False`.

Also, I think line 31 is wrong, but I cannot add a comment to it on that line. I suspect it should be `return merge(target, patch, inPlace);`

Reply:
Hmm... looking closer into the 
```    private Doc merge(Doc doc, Doc patch, Boolean inPlace = False)``` 
implementation I realized, that `inPlace` argument is not used for anything but a recursive call to itself. The actual value affects absolutely nothing.

Reply:
Not just that; the "inPlace" flag is not used for anything (e.g. ListMap construction)